cassandra/src/main/scala/filodb.cassandra/columnstore/PartitionKeysV2Table.scala

```diff
@@ -0,0 +1,219 @@
+package filodb.cassandra.columnstore
+
+import java.lang.{Integer => JInt, Long => JLong}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters.asScalaIteratorConverter
+
+import com.datastax.driver.core.{ConsistencyLevel, Row}
+import monix.eval.Task
+import monix.reactive.Observable
+
+import filodb.cassandra.FiloCassandraConnector
+import filodb.core.{DatasetRef, Response}
+import filodb.core.store.PartKeyRecord
+
+sealed class PartitionKeysV2Table(val dataset: DatasetRef,
+                                val connector: FiloCassandraConnector,
+                                writeConsistencyLevel: ConsistencyLevel)
```

Comment:
Will have to introduce read consistency similar to what got introduced in PR https://github.com/filodb/FiloDB/pull/1573

Reply:
Yes - This is done now after rebase.